docs/tour/adding-capabilities.mdx

```diff
@@ -16,10 +16,10 @@ Going from "Hello World" to a full-fledged application is simply a matter of ide
   <TabItem value="rust" label="Rust">
 
 :::info[Rust & WebAssembly]
-To perform the steps in this guide, you'll need to have a [Rust toolchain](https://www.rust-lang.org/tools/install) installed locally and the wasm32 target installed:
+To perform the steps in this guide, you'll need to have a [Rust toolchain](https://www.rust-lang.org/tools/install) installed locally and the wasm32-wasip1 target installed:
```

Comment:
We actually only want developers to run with wasm32-wasip2

Reply:
As soon as wasm32-wasip2 is in Rust stable, we can make this migration